invenio_app_rdm/theme/assets/templates/search/ResultsGridItemTemplate.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Card, Image } from 'semantic-ui-react';
+import _truncate from 'lodash/truncate';
+
+export function ResultsGridItemTemplate(result, index) {
+    return (
+      <Card fluid key={index} href={`#`}>
+        <Card.Content>
+          <Card.Header>{result.metadata.title}</Card.Header>
+          <Card.Description>
+            {_truncate(result.metadata.description, { length: 200 })}
```

Comment:
This changes with the new datamodel, just be aweare (or wait till its merged). It should be something like:

``` js
result.metadata.descriptions[0].description
```